### PR TITLE
GUI building entirely moved to python

### PR DESCRIPTION
**Pull request by DrunkSimurgh**
1. GUI building entirely moved to python (pyqt to be more precise).
2. This current setup is optimized to be able to run the first script (ColorMania); so, there are more works to do which follows.
3. "label", "floatedit", and "dropdown" elements from lua GUIs are converted to pyqt widgets (for "floatedit" element the conversion works for the ColorMania script but needs to be improved to better reflect its parent. Other elements of lua GUIs need to be converted to pyqt widgets.
4. Right now, the scripts will be applyed on all of the subtitle lines. This needs to be modified so that we can have the option to apply the scripts on the selected line(s) as well as filtering the lines based on some of their attributes (e.g. actor, style, etc.
~~5. The "settings.json" files need to have a "format" field so that we can identify in which environment the developers are presenting their GUIs' settings; for example in the ColorMania script, the setting.json file is structured based on lua GUIs format.~~

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
+  "DrunkSimurgh <saman.9267@gmail.com>",
+  "SSgumS <sssafais@hotmail.com>"
 ]
 description = "Python + Aegisub"
 name = "Pyegi"
 version = "0.0.1"
 
 [tool.poetry.dependencies]
 python = ">=3.9,<3.10"
+Pillow = "^9.0.1"
```
Comment (SSgumS) [inline pyproject.toml:12]: پیلو احتمالا درسته و برای لود گیف داری استفاده می‌کنی.
پایون‌اف‌ایکس ارتباطی به خود پایجی نداره.
پای کیو تی ۵ دیزاینر هم بعیده اصلا جزو دیپندنسی‌ها باشه.
بین پای‌کیوتی ۵ و ۶ هم منطقا یکیش الان داره استفاده می‌شه. پس یکیش اضافه‌ست.

**Comment by SSgumS** [comment]
> 1. GUI building entirely moved to python (pyqt to be more precise).
> 2. This current setup is optimized to be able to run the first script (ColorMania); so, there are more works to do which follows.
> 3. "label", "floatedit", and "dropdown" elements from lua GUIs are converted to pyqt widgets (for "floatedit" element the conversion works for the ColorMania script but needs to be improved to better reflect its parent. Other elements of lua GUIs need to be converted to pyqt widgets.
> 4. Right now, the scripts will be applyed on all of the subtitle lines. This needs to be modified so that we can have the option to apply the scripts on the selected line(s) as well as filtering the lines based on some of their attributes (e.g. actor, style, etc.
> 5. The "settings.json" files need to have a "format" field so that we can identify in which environment the developers are presenting their GUIs' settings; for example in the ColorMania script, the setting.json file is structured based on lua GUIs format.

I think passage 5 has not been implemented yet. So maybe you should strikethrough this passage as it's not done yet or implement it for Phase 1.

File: pyproject.toml (hunk 1 above)
Comment (DrunkSimurgh) [reply pyproject.toml:12]: ها اینا رو یادم رفت درست کنم. 
ولی بدیش اینه که دیزاینر رو نمی‌تونم حذف کنم. چون مدام دارم استفاده‌ش می‌کنم.
فقط باید یادم باشه مواقعی که دارم کامیت می‌زنم حذفش کنم.
بین پای‌کیوتی 5 و 6 هم فعلاً دارم با 6 جلو میرم و به مشکلی بر نخوردم. یحتمل 5 رو حذف کنم کلاً.

Comment (SSgumS) [reply pyproject.toml:12]: دیزاینر رو جزو دیپندنسی‌های اصلی نذار. بذار توی dev dependencyها که لازم نباشه پاک کنی.
با اضافه کردن آپشن -D توی کامند نصبش احتمالا به dev اضافه می‌شه.

**Comment by SSgumS** [comment]
Newer branches are available.

**Pull request closed without merging**